Makefile

```diff
@@ -32,7 +38,12 @@ ifeq ($(ARCH), x86_64)
 	endif
 endif
 
+ifeq ($(ARCH), aarch64)
```

Comment:
aarch64 + jit + cranelift should be working... is not that the case?

Reply:
Not yet! Last time I asked you about this, you told me we needed to upgrade to a newer cranelift to pick up their fixes to aarch64 support.

Here's a list of the current failures with make test-cranelift-jit:
```
failures:
    traps::start_trap_pretty
    traps::test_trap_stack_overflow
    traps::test_trap_trace
    traps::trap_display_multi_module
    traps::trap_display_pretty
    wast::cranelift::spec::align
    wast::cranelift::spec::memory_trap
    wast::cranelift::spec::simd::simd_bit_shift
    wast::cranelift::spec::simd::simd_boolean
    wast::cranelift::spec::simd::simd_const
    wast::cranelift::spec::simd::simd_i16x8_arith
    wast::cranelift::spec::simd::simd_i16x8_sat_arith
    wast::cranelift::spec::simd::simd_i32x4_arith
    wast::cranelift::spec::simd::simd_i64x2_arith
    wast::cranelift::spec::simd::simd_i8x16_arith
    wast::cranelift::spec::simd::simd_i8x16_sat_arith
    wast::cranelift::spec::simd::simd_lane
    wast::cranelift::spec::skip_stack_guard_page

test result: FAILED. 218 passed; 18 failed; 88 ignored; 0 measured; 0 filtered out
```

Reply:
Can we ignore this tests in cranelift in ARM meanwhile rather than not testing Clift+JIT in ARM?

Reply:
Done.